docs/releases/1.11.txt

```diff
@@ -168,7 +168,8 @@ Minor features
 Cache
 ~~~~~
 
-* ...
+* For memcached backends, the :setting:`LOCATION <CACHES-LOCATION>` setting
+  now supports defining multiple servers as a comma-delimited string.
```

Comment:
It would be helpful to add the motivation ("for convenience with some third-party services that use such strings in environment variables..." or similar).